docs/best_practice/crate_ami.txt

```diff
@@ -0,0 +1,114 @@
+.. highlight:: yaml
+.. _crate_ami:
+
+================================
+Launch a custom Crate AMI on AWS
+================================
+
+Crate provides custom AMIs (see `AMI`_) that can be launched from every
+EC2 user by using the AWS Management Console or `AWS CLI`_. The naming
+convention is as follows:
+
+  ``Crate-AWS-AMI-<VERSION>-<REVISION>``
+
+Find Crate AMI
+==================
+
+To launch an own EC2 instance based on the Crate AMI use the search panel to
+to find the image on the public AMI repository. After choosing the desired AMI
+by its Crate version it can be launched as an EC2 instance running on your
+account.
+
+.. image:: ../_static/crate-ami-search.png
+   :alt: Search for public Crate AMI on AWS Management Console
+   :width: 100%
+
+If you are using the `AWS CLI`_ you can find the shared Crate AMI by using the
+``describe-images`` command. The easiest way to search for it is by its
+``AMI Name``. AMIs are listed in JSON-format and sorted by its creation date.
+The following command lists every avaialble Crate AMI on the AWS platform.
+
+.. code-block:: bash
+
+    $ aws ec2 describe-images --filters "Name=name,Values=Crate-AWS-AMI-*"
+
+
+Launch the AMI
+==============
+
+A new EC2 instance can be started using the ``run-instances`` command. Use the
+available parameters of the command to configure its start-up behavior and
+hardware infrastructure.
+
+.. code-block:: bash
+
+    $ aws ec2 run-instances --image-id <AMI-ID> --count <NR-OF-INSTANCES> --instance-type <INSTANCE-TYPE> --user-data <USER-DATA>
+
+
+If you want to enable EC2 discovery on all instances to
```

Comment:
The AMI has Crate installed and configured so it uses EC2 discovery. This requires you to provide Crate with AWS credentials it can authenticate requests to the EC2 API.
You can provide these credentials (access key and secret key) as part of the user data when launching the instance.